.version

```diff
@@ -1 +1 @@
-0.75.0
+0.81.0
```

Comment:
Not sure if `ee` has it's own versioning scheme, but this is what it is in `main`